plugins/pulp_rpm/plugins/db/models.py

```diff
@@ -122,6 +126,7 @@ class NonMetadataPackage(UnitMixin, FileContentUnit):
     release = mongoengine.StringField(required=True)
     checksum = mongoengine.StringField(required=True)
```

Comment:
What are the meanings of `checksum` and `checksumtype` attributes with the introduction of `checksums`?